Source/CMakeLists.txt

```diff
@@ -48,7 +48,17 @@ set(SRCS ${DolphinProcessSrc}
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt6Widgets REQUIRED)
+if (WIN32)
+  find_package(Qt6Widgets QUIET)
+	if (NOT Qt6Widgets_FOUND)
+		message(STATUS "Qt package not found, using external lib")
+		list(APPEND CMAKE_PREFIX_PATH "..\\Externals\\Qt\\Qt6.5.3\\x64")
```

Comment:
Would this path be relative to the CMake file's location, or relative to the current working directory? We should aim for the former.

It may be a matter of prepending the path with `${CMAKE_CURRENT_LIST_DIR}\\`.